guides/03_building-with-blocks/04_custom-CSS-and-JS.md

```diff
@@ -51,9 +51,38 @@ with gr.Blocks(css=css) as demo:
 
 The CSS `#warning` ruleset will only target the second Textbox, while the `.feedback` ruleset will target both. Note that when targeting classes, you might need to put the `!important` selector to override the default Gradio styles.
 
-## Custom JS
+## Adding custom JavaScript to your demo
 
-Event listeners have a `_js` argument that can take a Javascript function as a string and treat it just like a Python event listener function. You can pass both a Javascript function and a Python function (in which case the Javascript function is run first) or only Javascript (and set the Python `fn` to `None`). Take a look at the code below:
+There are 3 ways to add javascript code to your Gradio demo:
 
+1. You can add javascript code as a string or as a filepath to the `js` parameter of the `Blocks` or `Interface` initializer. This will run the js code when the demo is first loaded.
```

Comment:
for consistency
```suggestion
1. You can add JavaScript code as a string or as a filepath to the `js` parameter of the `Blocks` or `Interface` initializer. This will run the js code when the demo is first loaded.
```